Monitor should not reallocate for an active reservation

super-cooper opened the pull request:
Reservation healing is skipped for leases which are active. Problematic hosts are still marked as unreservable.

super-cooper committed:
Monitor should not reallocate for an active reservation

Reservation healing is skipped for leases which are active. Problematic hosts are still marked as unreservable.

super-cooper commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Monitor should not reallocate for an active reservation](https://trello.com/c/LRRRmo2u/2705-monitor-should-not-reallocate-for-an-active-reservation)

Mark-Powers commented on blazar/plugins/monitor.py:
```diff
@@ -155,7 +156,7 @@ def heal_reservations(self, failed_resources, interval_begin,
                                                       resource_ids):
                 try:
                     if self._reallocate(allocation):
-                        if reservation['status'] == status.reservation.ACTIVE:
```
This branch can never run now, right? I think we should remove it

super-cooper replied:
Good catch

Mark-Powers reviewed (approved):
Looks good! I think we should remove the one section if it never will run though to simplify things.

super-cooper committed:
Remove active reservation branch after lease reallocation

We no longer reallocate on active leases, so there is no need to check to see if the lease was active after reallocating.
